share/commands/agent-diff

```diff
@@ -0,0 +1,76 @@
+#!/bin/sh
+# @description show diff between current file and the one before agent modification
+# @man This command will output file change in a diff format
+# @man +
+# @man *Options*:
+# @man +
+# @man *-l*: show diff from the given backup
+# @man +
+# @man *-n*: show diff from the nth backup before the last one
+# @man +
+# @man *-d*: show diff from a given date in the date command format (man date for details)
+# @man +
+# @man *filename*: the file to show diff from
+# @man +
+
+. "${BASEDIR}/../lib/common.sh"
+. "${BASEDIR}/../lib/cfengine_parsing.sh"
+
+# default logfile is last run
+canonify() {
+  echo -n "$1" | tr -c '[A-Za-z0-9_]' '_'
+}
+BASE_PATH="/var/rudder/modified-files/"
+BASE_NAME=""
+
+while getopts "l:n:d:" opt; do
+  case $opt in
+    l)
+      BASE_NAME="${OPTARG}"
+      ;;
+    n)
+      nth="${OPTARG}"
+      ;;
+    d)
+      timestamp=`LANG=C date -d "${OPTARG}" +"%s"`
```

Comment:
Then the -d option if rudder agent diff is not and cannot be supported for aix
rudder agent log has the same option